Makefile

```diff
@@ -35,3 +36,11 @@ publish: build ## Publish docker image
 clean: ## Clean autogenerated files/folders
 	@rm -rf bats
 	@rm -rf target
+
+create-release: ## Create github release given the APM Agent version if no tag release
+	@if [ -z "$(shell git tag -l $(AGENT_VERSION))" ]; then \
```

Comment:
I changed from git tag creation to gh release create, since it' already supported and working for the opbeans-java:
- https://github.com/elastic/opbeans-java/releases

Reply:
This condition will avoid running the gh create release if the tag has been already created